control/plugin/cpolicy/node.go

```diff
@@ -71,6 +71,56 @@ func NewPolicyNode() *ConfigPolicyNode {
 	}
 }
 
+func (c *ConfigPolicyNode) CopyRules() ([]Rule, error) {
+	rules := make([]Rule, len(c.rules))
+	i := 0
+	for _, rule := range c.rules {
```

Comment:
That's a good point! Do you think it'd be clearer without the i then and just using append.